example/src/NativeAdViewExample.tsx

```diff
@@ -110,7 +61,50 @@ export const NativeAdViewExample = ({
             />
             {isNativeAdShowing && (
                 <View style={styles.container}>
-                    <NativeAdExample />
+                    <NativeAdView
+                        adUnitId={adUnitId}
+                        placement="myplacement"
+                        customData="mycustomdata"
+                        ref={nativeAdViewRef}
+                        style={styles.nativead}
+                        onAdLoaded={(adInfo: AdInfo) => {
+                            if (adInfo?.nativeAd?.mediaContentAspectRatio) {
+                                setAspectRatio(adInfo?.nativeAd?.mediaContentAspectRatio);
+                            }
+                            log('Native ad loaded from ' + adInfo.networkName);
+                            setIsNativeAdLoading(false);
+                        }}
+                        onAdLoadFailed={(errorInfo: AdLoadFailedInfo) => {
+                            log(
+                                'Native ad failed to load with error code ' +
+                                    errorInfo.code +
+                                    ' and message: ' +
+                                    errorInfo.message
+                            );
```

Comment:
If we can easily avoid it, a log message should not cost 6 lines of code

Reply:
This will be fixed when we merge the preload PR since we updated `printWidth` for the `prettier` command.